packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -769,6 +766,13 @@ export class ContainerRuntime extends EventEmitter
             this.setNewContext(key, remoteContext);
         }
 
+        this.blobManager = new BlobManager(
+            this.IFluidHandleContext,
+            () => this.storage,
+            // (blobId) => this.submit(ContainerMessageType.BlobAttach, blobId),
+            (blobId) => this.context.submitFn(MessageType.BlobAttach, undefined, false, { blobId }),
```

Comment:
I'd put no-op handler here for now and come back to supplying metadata in next PR

Reply:
Actually, I'll take it back - it useful from day one to have an op, such that unit tests can text it :)
I'd change it to ContainerMessageType op, as that's what we will use anyway.